mlir/lib/Conversion/MIGraphXToTosa/MIGraphXToTosa.cpp

```diff
@@ -57,6 +58,19 @@ static TosaOp createOpAndInfer(mlir::PatternRewriter &rewriter,
   return op;
 }
 
+static tosa::CastOp createCastOp(mlir::PatternRewriter &rewriter,
```

Comment:
I know it became house style in this file somehow, but can we lose the leading `mlir::`s here?

Reply:
Good advice!